Description: Fix build error by adding -fPIC and importing CFLAGS from the enviornment
Author: Jacob Adams <tookmund@gmail.com>
Forwarded: yes
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-CFLAGS = -DSHAPE -Wall -Werror
+CFLAGS := $(CFLAGS) -DSHAPE -Wall -Werror -fPIC
 LDLIBS = -lXext -lX11
 BIN = $(DESTDIR)/usr/bin/
 
